apps/emqx_bridge_bigquery/src/emqx_bridge_bigquery_connector_schema.erl

```diff
@@ -0,0 +1,168 @@
+fields(connector_config) ->
+    [
+        {connect_timeout,
+            mk(
+                emqx_schema:timeout_duration_ms(),
+                #{
+                    default => <<"15s">>,
+                    desc => ?DESC(emqx_bridge_gcp_pubsub, "connect_timeout")
+                }
+            )},
+        {pool_size,
+            mk(
+                pos_integer(),
+                #{
+                    default => 8,
+                    desc => ?DESC(emqx_bridge_gcp_pubsub, "pool_size")
+                }
+            )},
+        {pipelining,
+            mk(
+                pos_integer(),
+                #{
+                    default => 100,
+                    desc => ?DESC(emqx_bridge_gcp_pubsub, "pipelining")
+                }
+            )},
+        {max_retries,
+            mk(
+                non_neg_integer(),
+                #{
+                    required => false,
+                    default => 2,
+                    desc => ?DESC(emqx_bridge_gcp_pubsub, "max_retries")
+                }
+            )},
+        emqx_connector_schema:ehttpc_max_inactive_sc(),
+        {service_account_json,
+            mk(
+                binary(),
+                #{
+                    required => true,
+                    validator => fun emqx_bridge_gcp_pubsub:service_account_json_validator/1,
+                    sensitive => true,
+                    desc => ?DESC(emqx_bridge_gcp_pubsub, "service_account_json")
+                }
+            )}
+    ] ++
+        emqx_connector_schema:resource_opts().
+
+desc("config_connector") ->
+    ?DESC("config_connector");
+desc(_Name) ->
+    undefined.
+
+%%------------------------------------------------------------------------------
+%% `emqx_connector_examples' API
+%%------------------------------------------------------------------------------
+
+connector_examples(Method) ->
+    [
+        #{
+            <<"bigquery">> => #{
+                summary => <<"BigQuery Connector">>,
+                value => connector_example(Method)
+            }
+        }
+    ].
+
+connector_example(get) ->
+    maps:merge(
+        connector_example(put),
+        #{
+            status => <<"connected">>,
+            node_status => [
+                #{
+                    node => <<"emqx@localhost">>,
+                    status => <<"connected">>
+                }
+            ]
+        }
+    );
+connector_example(post) ->
+    maps:merge(
+        connector_example(put),
+        #{
+            type => atom_to_binary(?CONNECTOR_TYPE),
+            name => <<"my_connector">>
+        }
+    );
+connector_example(put) ->
+    #{
+        enable => true,
+        description => <<"My connector">>,
+        connect_timeout => <<"15s">>,
+        max_inactive => <<"10s">>,
+        max_retries => 2,
+        pipelining => 100,
+        pool_size => 8,
+        service_account_json =>
+            <<"{\"client_email\": \"test@myproject.iam.gserviceaccount.com\", ...}">>,
```

Comment:
Shouldn't that be a correct json?

Nit:

m.b. use sigils for a simpler binary literal without much escaping:
```erlang
service_account_json => ~b'{"client_email": "test@myproject.iam.gserviceaccount.com", ...}',
```

Reply:
I don't think so: this is just an example, this whole JSON is a credential, and it's quite big.

But the sigil idea is quite nice: I didn't know that Erlang had them now.  :sweat_smile: :see_no_evil: